config/dev/openstack-credentials.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openstack-cloud-config
+  namespace: ${NAMESPACE}
+stringData:
+  clouds.yaml: |
+    clouds:
+      openstack:
+        auth:
+          auth_url: ${OS_AUTH_URL}
+          username: ${OS_USERNAME}
+          password: ${OS_PASSWORD}
+          project_id: ${OS_PROJECT_ID}
+          project_name: ${OS_PROJECT_NAME}
```

Comment:
AFAIR, the `project_name` is optional. Will this config be properly rendered if the `${OS_PROJECT_NAME}` is unset?